draft-ietf-mmusic-ice-sip-sdp-38.xml

```diff
@@ -231,6 +238,12 @@ check Binding requests on those candidates.
         </section>
         <section anchor="sec-ice-mismatch" title="Verifying ICE Support Procedures">
           <t>
+            An ICE agent is considered to indicate support of ICE by including
+            at least the SDP ice-pwd and ice-ufrag attributes in an offer or answer.
+            An ICE agent compliant will this specification also includes an 
+            SDP ice-options attribute with an "ice2" attribute value.
```

Comment:
typo will -> with.
I suggest change
"An ICE agent compliant will this specification also includes an " to "An ICE agent compliant will this specification MUST also include an " to make it normative ?

Reply:
Will fix as suggested (s/will this/with this)